common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,53 +1,60 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
-    <fieldset role="${input_type}group" aria-label="${label}">
-        % for choice_id, choice_description in choices:
-        <label for="input_${id}_${choice_id}"
-            ## If the student has selected this choice...
-            % if input_type == 'radio' and ( (isinstance(value, basestring) and (choice_id == value))  or  (not isinstance(value, basestring) and choice_id in value) ):
+    <fieldset>
+      % if response_data and response_data['label']:
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+      % else:
+        <legend>Question</legend>
```

Comment:
ok, well, the discussion about how best to present the text continues, and new viewpoints keep getting added... Unfortunately now I can't estimate when we'll have new test for you, @ehteshamkafeel